python/ml4ir/base/config/parse_args.py

```diff
@@ -136,6 +129,15 @@ def define_args(self):
             help="A space separated list of metrics to compute.",
         )
 
+        self.add_argument(
```

Comment:
Not directly related, but I counted 46 add_argument in this file, should we consider moving this arguments to a conf file at some points?